Cargo.toml

```diff
@@ -43,10 +43,11 @@ chrono = "0.4.19"
 tempfile = "3"
 home = { version = "0.5.5", optional = true }
 semver = "1.0.18"
-ics23 = "0.10.2"
-cosmos-sdk-proto = {version = "0.19.0", optional = true }
+ics23 = "0.12.0"
+cosmos-sdk-proto = {version = "0.23.0", optional = true }
 prometheus_exporter = "0.8.5"
 lazy_static = "1.4.0"
+getrandom = {version = "0.2.15", features = ["js"] }
```

Comment:
Why was this added? I don't see it being used.

Reply:
Using this to force the dependency tree to include the js feature in getrandom so that wasm builds.

Reply:
Shouldn't that be added to the `wasm` crate then?

Reply:
Good point. Fixed.